### PR TITLE
add 58. Length of Last Word

### PR DESCRIPTION
**Pull request by katsurada1**
Please check my code.

### CODE REVIEW

File: 58.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+            s (str): A given a string consisting of some words separated by some number of spaces.
+        Returns:
+            int: The length of the last word consisting of non-space characters in the string.
+        """
+        
+        l = s.split()
+        if l:
+            return len(l[-1])
+        
+        else:
```
Comment (kobakobashu) [inline 58.py:18]: Good consideration.
However, error handling should be done first.

if not l:
    return 0

return len(l[-1])

**Review by kobakobashu — CHANGES_REQUESTED** [review]
👀

**Comment by katsurada1** [comment]
I modified the code.
Please check my code.

**Review by kobakobashu — APPROVED** [review]
LGTM

**Pull request merged**